### DISCUSSION TITLE
How to add 'Github repo' link in publications?

### DISCUSSION THREAD

**Discussion by dherrera1911** [academicpages/academicpages.github.io#2311] [Q&A]
For my publications, I often have a Github repo associated that I would like to display in the publications list, like the link 'Slides' or 'Download paper' do. I'm trying to figure out by looking at the markdown_generator/ contents how I could either add a tab named 'Github repo' linking to a 'githuburl' defined in the publications yaml files, or to change the name of 'Slides' into Github repo, but I can't figure this out.

**Comment by rjzupkoii** [comment]
This requires you to modify the template a bit. For the Markdown pages for the publications you would need to add the following:

```Markdown
---
title: "Your Publication title"

# Omitting the other existing entries

repository: '[http://](https://github.com/academicpages/academicpages.github.io)'
---
```

Then under `_includes/archive-single.html` you would need to add the following:

```liquid
  {% if post.repository %}
    <a href="{{ post.repository }}">GitHub Repository</a>
  {% endif %}      
```
It can take a bit of effort to get things styled nicely, but I did something similar for my site to include a link to download papers.
